tests/fixtures/inversify-dynamic-container/ioc.ts

```diff
@@ -0,0 +1,15 @@
+import { Container } from 'inversify';
+import { ManagedController } from './managedController';
+import { ManagedService } from './managedService';
+
+export let containerMethodCalled = false;
+
+const iocContainer = function (request: Express.Request): Container {
```

Comment:
I think we should use the request and check that the request is actually used.

Reply:
Sure! What were you thinking? Using the request to resolve a container or just a dummy test to ensure request is passed correctly?

Reply:
Don't bind the ManagedService as a singleton, let the request be injected the req into the constructor, return the request's path from a getPath() method (instead of getModel). Something like that. Doesn't have to be the greatest use ever, but should show the idea and the funcitonality

Reply:
I've updated the test to inject the request path.